css-four-column-layout/styles.css

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+  font-family: Barlow, sans-serif;
+  text-align: center;
+}
+
+body {
+  margin: 0;
+  background-color: #f7f8fa;
+}
+
+h2 {
+  font-size: 1.1rem;
+  font-weight: 800;
+  margin-top: 0.5rem;
+  margin-bottom: 0.6rem;
+}
+
+h3 {
+  font-size: 0.62rem;
+  font-weight: 800;
+  margin-bottom: 1.2rem;
+}
+
+p {
+  font-size: 0.52rem;
+  font-weight: 500;
+}
+
+.container {
+  width: 850px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.col-1-4 {
+  margin: 0 -2.25rem;
+}
```

Comment:
`column` classes should only have a `flex-basis` or `width`, and nothing else. You should apply this `margin` on a separate class that you can add in addition to the `column` class, and you should also set an appropriate size on this `column` class.